### PR TITLE
Implement OnTimeout

### PR DESCRIPTION
**Pull request by bonewell**

### CODE REVIEW

**Comment by bonewell** [comment]
@KKarlash 
@DKlimenko 
@VladSemenyuk 
Please, review!

File: src/components/vr_module/src/convertor.cc [hunk 1]
```diff
@@ -46,6 +46,8 @@ inline vr_hmi_api::ResultCode Convertor(vr_mobile_api::ResultCode value) {
       return vr_hmi_api::REJECTED;
     case vr_mobile_api::WARNINNG:
       return vr_hmi_api::WARNINGS;
+    case vr_mobile_api::TIMEOUT:
+          return vr_hmi_api::TIMEOUT;
```
Comment (KKarlash) [inline src/components/vr_module/src/convertor.cc]: align

File: src/components/vr_module/src/convertor.cc [hunk 2]
```diff
@@ -63,6 +65,8 @@ inline vr_mobile_api::ResultCode Convertor(vr_hmi_api::ResultCode value) {
       return vr_mobile_api::WARNINNG;
     case vr_hmi_api::REJECTED:
       return vr_mobile_api::REJECTED;
+    case vr_hmi_api::TIMEOUT:
+          return vr_mobile_api::TIMEOUT;
```
Comment (KKarlash) [inline src/components/vr_module/src/convertor.cc]: align

**Comment by KKarlash** [comment]
Reviewed

File: src/components/vr_module/src/convertor.cc [hunk 3]
```diff
@@ -46,6 +46,8 @@ inline vr_hmi_api::ResultCode Convertor(vr_mobile_api::ResultCode value) {
       return vr_hmi_api::REJECTED;
     case vr_mobile_api::WARNINNG:
       return vr_hmi_api::WARNINGS;
+    case vr_mobile_api::TIMEOUT:
```
Comment (DKlimenko) [inline src/components/vr_module/src/convertor.cc]: please add bracket { for each case

File: src/components/vr_module/src/convertor.cc [hunk 4]
```diff
@@ -63,6 +65,8 @@ inline vr_mobile_api::ResultCode Convertor(vr_hmi_api::ResultCode value) {
       return vr_mobile_api::WARNINNG;
     case vr_hmi_api::REJECTED:
       return vr_mobile_api::REJECTED;
+    case vr_hmi_api::TIMEOUT:
```
Comment (DKlimenko) [inline src/components/vr_module/src/convertor.cc]: please add bracket { for each case

**Comment by DKlimenko** [comment]
reviewed

File: src/components/vr_module/src/convertor.cc (hunk 3 above)
Comment (bonewell) [inline src/components/vr_module/src/convertor.cc]: I can add, but I don't understand why?

File: src/components/vr_module/src/commands/process_data.cc [hunk 5]
```diff
@@ -80,7 +80,13 @@ bool ProcessData::Execute() {
...
 void ProcessData::OnTimeout() {
   LOG4CXX_AUTO_TRACE(logger_);
-  // TODO(KKolodiy): add timeout processing
+  message_.set_rpc_type(vr_hmi_api::RESPONSE);
+  vr_hmi_api::ProcessDataResponse hmi_response;
+  hmi_response.set_result(vr_hmi_api::TIMEOUT);
+  std::string params;
+  hmi_response.SerializeToString(&params);
+  message_.set_params(params);
+  module_->SendToHmi(message_);
```
Comment (VladSemenyuk) [inline src/components/vr_module/src/commands/process_data.cc]: Duplicated code

**Comment by VladSemenyuk** [comment]
Reviewed

**Pull request merged**